src/cursor.rs

```diff
@@ -469,9 +469,12 @@ impl Cursor {
     /// # Return value
     ///
     /// Returns a vector containing the BSON documents that were read.
-    pub fn next_batch(&mut self) -> Result<Vec<bson::Document>> {
-        let n = self.batch_size;
-        self.next_n(n)
```

Comment:
Fair enough, although I'd be in favor of getting rid of this at some point in the future.